TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -905,6 +909,128 @@ private void WritePeptideResults()
+            List<PeptideSpectralMatch> modifiedVariantPeptides = confidentVariantPeps.Where(p => p.ModsIdentified.Count() > 0).ToList();
+            List<PeptideSpectralMatch> modifiedVariantSitePeptides = new List<PeptideSpectralMatch>();
+            foreach (var entry in modifiedVariantPeptides)
+            {
+                var variants = entry.BestMatchingPeptides.SelectMany(p => p.Peptide.Protein.AppliedSequenceVariations.Where(v => PsmTsvWriter.IntersectsWithVariation(p.Peptide, v, true) ==true).ToList()).ToList();
+                var mods = entry.ModsIdentified;
+                bool modifiedVariant = false;
+                foreach (var mod in mods)
+                {
+                    int residue = mod.Value;                    
+                    var modOnVariant = variants.Where(p => p.OneBasedBeginPosition >= residue && p.OneBasedEndPosition <= residue);
+                    if (modOnVariant.Count() > 0)
+                    {
+                        modifiedVariant = true;
+                    }
+                }
+                if (modifiedVariant == true)
+                {
+                    modifiedVariantSitePeptides.Add(entry);
+                }
+            }
+            foreach (var peptide in confidentVariantPeps)
+            {
+                var variantPWSM = peptide.BestMatchingPeptides.FirstOrDefault();
+                var variants = variantPWSM.Peptide.Protein.AppliedSequenceVariations;
+                foreach (var variant in variants)
+                {
+                    bool intersects = PsmTsvWriter.IntersectsWithVariation(variantPWSM.Peptide, variant, true);
+                    if(intersects == true)
+                    {
+                        if (variant.Description.Description.Contains("missense_variant"))
+                        {
+                            savCount++;
+                        }
+                        if (variant.Description.Description.Contains("frameshift_variant"))
+                        {
+                            frameshiftCount++;
+                        }
+                        if (variant.Description.Description.Contains("stop_gained"))
+                        {
+                            stopGainCount++;
+                        }
+                        if (variant.Description.Description.Contains("conservative_inframe_insertion") || variant.Description.Description.Contains("disruptive_inframe_deletion"))
+                        {
+                            indelCount++;                            
+                        }
+                        
+                    }
+                }
+            }
+
+            string[] variantResults = new string[12];
+            variantResults[0] = "Variant Result Summary";
+            variantResults[2] = "--------------------------------------------------";
+            variantResults[4] = "Number of ambiguous variant peptides at 1% FDR: " + variantPeptides.Count();
```

Comment:
`Number of identified variant peptides ...`

Reply:
There are potential variant peptides. Their base sequence could result from either a variant containing protein or a canonical protein sequence. I will try to figure out a better way to convey this. But it doesn't feel right to me to just call them identified variant peptides.

Reply:
That makes sense. I guess `potential variant peptides` makes more sense than `ambiguous`